data/test_yaml_stories/stories.yml

```diff
@@ -0,0 +1,50 @@
+stories:
+- story: simple_story_without_checkpoint
+  steps:
+  - user: /simple
+  - action: utter_default
+  - action: utter_greet
+
+- story: simple_story_with_only_start
+  steps:
+  - checkpoint: check_greet # checkpoints at the start define entry points
+  - user: /simple
+  - action: utter_default
+
+- story: simple_story_with_only_end
+  steps:
+  - user: /hello
+  - action: utter_greet
+  - slot: name
+    value: peter
+  - checkpoint: check_greet # checkpoint defining the end of this turn
+
+- story: simple_story_with_multiple_turns
+  steps:
+  - or:
+    - user: /affirm
+    - user: /thank_you
+  - action: utter_default
+  - user: /goodbye
+  - action: utter_goodbye
+  - checkpoint: check_goodbye
+
+- story: why does the user want to leave?
+  steps:
+  - checkpoint: check_goodbye
+  - user: /why
+  - action: utter_default
+  - checkpoint: check_greet
+
+- story: show_it_all
+  steps:
+  - checkpoint: check_greet
+  - checkpoint: check_hello # allows multiple entry points
```

Comment:
do we actually need checkpoints with yaml anchors?

Reply:
I think we do, at least for now it's the least confusing option for the users migrating from the MD.